### PR TITLE
[WIP] MINT-1133 added NEM token to the address list

### PR DESCRIPTION
**Pull request by e-maximus**
added NEM token to the address list

### CODE REVIEW

File: src/redux/session/selectors.js [hunk 1]
```diff
@@ -39,6 +40,27 @@ export const BALANCES_COMPARATOR_URGENCY = (item1, item2) => {
...
+      ]
+      const newTitle = titleList.find((s) => s.symbol === symbol)
+      return newTitle?newTitle.title:symbol
+    }
+
+    const addressesInWallet = mainWallet.addresses()
+    return rebuildProfileTokens(profile, tokens)
+      .filter((token) => PROFILE_PANEL_TOKENS.includes(token.symbol))
+      .map((token) => {
+        return { ...token, title: symbolToTitle(token.symbol), address: addressesInWallet.item(token.blockchain).address() }
```
Comment (ipavlenko) [inline src/redux/session/selectors.js]: Bad idea
Our TokenModel is an immutable model.
Pass here an object with a new title & reference to TokenModel

File: src/redux/session/selectors.js [hunk 2]
```diff
@@ -39,6 +40,27 @@ export const BALANCES_COMPARATOR_URGENCY = (item1, item2) => {
...
+    const symbolToTitle = (symbol) => {
+      const titleList = [
+        { symbol: 'XEM', title: 'NEM' },
+      ]
+      const newTitle = titleList.find((s) => s.symbol === symbol)
+      return newTitle?newTitle.title:symbol
+    }
+
+    const addressesInWallet = mainWallet.addresses()
+    return rebuildProfileTokens(profile, tokens)
```
Comment (ipavlenko) [inline src/redux/session/selectors.js]: Sometimes we may have no XEM or BTC token initialized but need a NEM or BTC address.
Please just hardcode ETH, BTC, BTG & NEM here.

Comment (ipavlenko) [reply src/redux/session/selectors.js]: For example:
1) BTC or XEM middleware unavailable
2) BTC or XEM switched off from the user interface (hidden from profile, hidden from gallery).

You have in global scope:
EthereumDAO
NemDAO
BitcoinDAO (BTC)
BitcoinDAO (BTG)
it is all you need to get addresses & keys.

**Comment by e-maximus** [comment]
Closed because of duplicate changes
see https://github.com/ChronoBank/ChronoMint/pull/568

**Pull request closed without merging**